overview/default.nix

```diff
@@ -7,28 +8,32 @@
   inherit
     (builtins)
     any
+    attrNames
     attrValues
     concatStringsSep
     filter
     isList
-    mapAttrs
-    match
     readFile
+    stringLength
     substring
     toJSON
     toString
     ;
 
   inherit
-    (pkgs.lib)
+    (lib)
     concatLines
+    flattenAttrsDot
+    flip
     mapAttrsToList
     optionalString
-    unique
     ;
 
   empty = xs: assert isList xs; xs == [];
   heading = i: text: "<h${toString i}>${text}</h${toString i}>";
+  isPrefix = prefix: candidate: prefix == substring 0 (stringLength prefix) candidate;
```

Comment:
And I thought we have that in Nixpkgs?

Reply:
Ah, it's just called `hasPrefix`. Will fix.

Reply:
Fixed in 70146c7.